### ISSUE TITLE
interpretation of scores in B.5 - Table 10

### ISSUE THREAD

**Issue by adrienchaton** [OATML-Markslab/Tranception#7]
Dear authors,

Thanks for the paper and code release, very interesting things !
One thing that shook me at the end of reading is the performance being similar for alpha = 0 or 1 in B.5 - Table 10.
If I follow correctly, this would mean that the scoring derived from MSA alone or derived from the language model alone would be about equally good at assessing fitness ? (at least in the general case where MSA of reasonable depth may be computed)

As a DL enthusiast, this observation, if right, is quite challenging : )

**Comment by pascalnotin** [comment]
Dear Adrien,

You understanding is correct -- the two modes of inference have roughly the same ability to predict protein fitness. Another way to see this is that the optimal value of the mixing parameter (alpha) between the two inference modes that is selected on the validation set is 0.6 (ie. the model leans slightly more on retrieval inference than on autoregressive inference).

The two modes are quite complementary though (as they look at the fitness prediction task from different angles) which is leading to the SOTA performance on the ProteinGym benchmarks. To my knowledge, there is no modeling approach at present that comes close to that without relying on MSAs somewhere. With that said, there has been very interesting progress over the past few months to do away with MSAs for 3D structure prediction, so perhaps we will see similar trends for fitness prediction in the near future.

**Comment by adrienchaton** [comment]
Thanks Pascal for confirming.

MSA is a very strong baselines since it relies on sequences selected by evolution, thus naturally fit and a very good proxy to "fitness".
I am far from knowledgeable into MSA algorithms but one of the promise of language models is to go beyond sequence similarity, that is what is used in MSA algorithms.
However the conclusion we observe at the moment is that SOTA LMs cannot outperform MSAs based on low-level sequence similarity, however we invest very large compute in both training and inference (and we provide also huge amount of data for training).

There is more to it and looking at whole average results is not enough, but as a simple take-away it challenges a lot the current state of our DL methods.

**Comment by adrienchaton** [comment]
One direction which may brings us further is to incorporate together sequence models and structure models, which is already being explored in e.g. LM-GVP.
But at the moment, structure prediction has not been proven to work well for all proteins (whether it uses MSA or not), neither to be sensitive to single point mutations.

The breakthrough is still to come : )

**Comment by pascalnotin** [comment]
Retrieval of MSAs was one way for us to implicitly leverage information about the 3D structure of a given protein family, but very much agree with you on the potential from leveraging the structure more explicitly ;)
